adapter/test/CMakeLists.txt

```diff
@@ -48,4 +48,16 @@ endfunction()
 
 
 add_subdirectory(${CMAKE_CURRENT_SOURCE_DIR}/posix)
-add_subdirectory(${CMAKE_CURRENT_SOURCE_DIR}/stdio)
\ No newline at end of file
+add_subdirectory(${CMAKE_CURRENT_SOURCE_DIR}/stdio)
+
+if(HERMES_INSTALL_TESTS)
```

Comment:
Shouldn't the daemon be installed regardless of whether `HERMES_INSTALL_TESTS` is enabled?

Reply:
I was not sure. I thought we would make a daemon eventually probably more capable than the simple daemon for the tests.

Reply:
Added to install even if HERMES_INSTALL_TESTS is disabled.